galaxy/api/v2/serializers/collection.py

```diff
@@ -65,7 +177,7 @@ def get_imported_version(self, obj):
             return None
         return {
             'id': obj.imported_version.pk,
-            'href': reverse('api:v2:collection-version-detail',
+            'href': reverse('api:v2:version-detail-by-id',
```

Comment:
AFAIK `by-id` suffix is not necessary, you may have multiple routes with same name, but different parameters.